### PR TITLE
Allow user to define harness in ks_meta multiple times

### PR DESCRIPTION
**Pull request by StykMartin**
Following definition should be considered as valid
```XML
<recipe ks_meta="harness=restraint-rhts-0.1.44 harness='restraint-rhts beakerlib-redhat' ">
```

Bug: [1803555](https://bugzilla.redhat.com/show_bug.cgi?id=1803555)
It is expected that we will merge all items and make DNF resolve the situation

### CODE REVIEW

File: IntegrationTests/src/bkr/inttest/server/test_kickstart.py [hunk 1]
```diff
@@ -4188,3 +4188,24 @@ def test_kickstart_overflow(self):
...
+                        </distroRequires>
+                        <hostRequires/>
+                        <task name="/distribution/check-install" />
+                    </recipe>
+                </recipeSet>
+            </job>
+            ''', self.system)
+        ks = recipe.installation.rendered_kickstart.kickstart
+        ks_lines = ks.splitlines()
+        self.assert_('$package_command -y install great-restraint dead-beah' in ks_lines, ks)
```
Comment (ernestask) [inline IntegrationTests/src/bkr/inttest/server/test_kickstart.py:4211]: What about `assertIn()` without splitting?

Comment (StykMartin) [reply IntegrationTests/src/bkr/inttest/server/test_kickstart.py:4211]: I would like to make sure that content is exactly same on the given line.

**Comment by cbouchar** [comment]
I don't understand the importance of this.  Additionally perhaps you should document how this will be resolved.  Which takes precedence?

**Comment by StykMartin** [comment]
There is no need to document precedence. All variables are treated in Beaker in the same manner as before and I didn't touch this part.
You will define it in XML -> You will get same content in ks_meta internally

**Pull request merged**